### ISSUE TITLE
Help please Outputing a specific Channel

### ISSUE THREAD

**Issue by timmassey** [mtongnz/espDMX#5]
Evening

thank you for sharing espDMX, I have been looking at the dmxChase example and I have that working within another project I am working on and it works perfectly, but my what I can seem to so if get a artnet values from channel one and then turn that into a DMX value to output.

Any help would be gratefully received.

Thank you in advanced

Tim

**Comment by netmindz** [comment]
Yeah, I'm equally trying to move from different DMX library to yours in the hope he output is more reliable but the lack of a simple setChannel call makes this harder

**Comment by mtongnz** [comment]
Use the getChans function to get a pointer to the data array. Then you can set individual channels
